src/applications/static-pages/homepage/tests/new-homepage.cypress.spec.js

```diff
@@ -0,0 +1,459 @@
+        );
+        checkLinkNavigation(
+          linksListSelector,
+          'Review or update your dependents',
+          '/view-change-dependents',
+        );
+        checkLinkNavigation(
+          linksListSelector,
+          'Get reimbursed for travel pay',
+          '/health-care/get-reimbursed-for-travel-pay/',
+        );
+        checkLinkNavigation(
+          linksListSelector,
+          'Get your VA medical records',
+          '/health-care/get-medical-records/',
+        );
+      });
+    });
+  });
+
+  describe('Blog/News section', () => {
+    describe('Pathfinder article', () => {
+      // eslint-disable-next-line @department-of-veterans-affairs/axe-check-required
+      it('loads blog/news-section content - C32638', () => {
+        // skipping page-content AXE-check -- already done in first test
+        cy.get('.homepage-blog')
+          .should('exist')
+          .scrollIntoView()
+          .within(() => {
+            cy.get('h3#pathfinder-the-front-door-for-')
+              .should('exist')
+              .next('p')
+              .should('exist')
+              .and('not.be.empty');
+          });
+      });
+
+      // eslint-disable-next-line @department-of-veterans-affairs/axe-check-required
+      it('navigates to link-destinations - C32639', () => {
+        // skipping page-content AXE-check -- already done in first test
+        checkLinkNavigation(
+          '.homepage-blog',
+          'Pathfinder: The front door for engaging with VA',
+          'https://pathfinder.va.gov/',
+        );
+        checkLinkNavigation(
+          '.homepage-blog',
+          'Read the full article',
+          'https://pathfinder.va.gov/',
+        );
+        checkLinkNavigation(
+          '.homepage-blog',
+          'More VA news ',
+          'https://news.va.gov/',
+        );
+      });
+    });
+  });
+
+  describe('Benefit hubs section', () => {
+    // eslint-disable-next-line @department-of-veterans-affairs/axe-check-required
+    it('loads benefit-hubs section content - C32640', () => {
+      const checkHubContents = h3Id => {
+        cy.get(`.homepage-benefits-row [data-e2e="hub"] h3#${h3Id}`)
+          .should('exist')
+          .next('p')
+          .should('exist')
+          .and('not.be.empty');
+      };
+
+      // skipping page-content AXE-check -- already done in first test
+      cy.get('[data-e2e="hub"]').should('have.length.at.least', 11);
+      checkHubContents('service-member-benefits');
+      checkHubContents('family-member-benefits');
+      checkHubContents('burials-and-memorials');
+      checkHubContents('careers-and-employment');
+      checkHubContents('housing-assistance');
+      checkHubContents('pension');
+      checkHubContents('life-insurance');
+      checkHubContents('education-and-training');
+      checkHubContents('records');
+      checkHubContents('health-care');
+      checkHubContents('disability');
```

Comment:
Dynamic content assertion. These may change, but typically far less often. There is also another one to be added in the near future called VA Department Information.